ui/src/app/pages/profile/profile-page.spec.tsx

```diff
@@ -148,11 +148,30 @@ describe('ProfilePageComponent', () => {
 
     const wrapper = component();
 
-    let streetAddress1 = wrapper.find('[data-test-id="streetAddress1"]').first();
+    const streetAddress1 = wrapper.find('[data-test-id="streetAddress1"]').first();
     expect(streetAddress1.prop('value')).toBe('Main street');
 
     streetAddress1.simulate('change', {target: {value: ''}});
 
     expect(getSaveProfileButton(wrapper).first().prop('disabled')).toBe(true);
   });
+
+  it('should not display controlled tier card when there is no controlled tier', async() => {
+    const wrapper = component();
+    const profileCardCompleteButtons = wrapper.find('[data-test-id="incomplete-button"]');
```

Comment:
Are these test IDs exclusive to the CT Training module?  The other cards can also be in a complete/incomplete state too, right?

Reply:
The ID's are not exclusive to the CT training module. Yes, they can be in an incomplete state - the mock data has them setup that way, that is why I am checking for 4 of them to be present.